pep-0703.rst

```diff
@@ -0,0 +1,1691 @@
+   Paul E. McKenney. 2022.
+   https://mirrors.edge.kernel.org/pub/linux/kernel/people/paulmck/perfbook/perfbook.html.
+
+.. [#typesafe_rcu] ``SLAB_TYPESAFE_BY_RCU`` is an example in which RCU
+   protects blocks of memory and not any individual object.  See
+   https://www.kernel.org/doc/html/latest/RCU/whatisRCU.html#analogy-with-reference-counting.
+
+.. [#hotspotgc] "HotSpot Virtual Machine Garbage Collection Tuning Guide."
+   https://docs.oracle.com/en/java/javase/12/gctuning/hotspot-virtual-machine-garbage-collection-tuning-guide.pdf.
+   Most of the hotspot garbage collectors are generational, with the
+   notable exception of ZGC, although there is ongoing work to make
+   that generational.
+
+.. [#decapo] `The DaCapo Benchmarks: Java Benchmarking Development and
+   Analysis
+   <https://openresearch-repository.anu.edu.au/bitstream/1885/33723/2/01_Blackburn_The_DaCapo_Benchmarks:_Java_2006.pdf>`_.
+   See column "Nursery Survival" in Table 4.
+
+.. [#exploitingmemoryjava] "Exploiting memory usage patterns to improve garbage collections in Java."
+   https://dl.acm.org/doi/abs/10.1145/1852761.1852768.
+
+.. [#cpythongc] "most things usually turn out to be reachable"
+    https://github.com/python/cpython/blob/cd6655a8589e99ae4088b3bed4a692a19ed48779/Modules/gcmodule.c#L1106.
+
+.. [#golangc] The Go team observed something similar in Go, but due to
+   escape analysis and pass-by-value instead of reference
+   counting. Recent versions of Go use a non-generational garbage
+   collector. https://go.dev/blog/ismmkeynote.
+
+.. [#nogil] https://github.com/colesbury/nogil.
+
+.. [#howto] Python HOWTOs.
+   https://docs.python.org/3/howto/index.html.
+
+.. [#pep659] :pep:`659` – Specializing Adaptive Interpreter.
+
+.. [#gilectomy] Gilectomy.
+   Larry Hastings.
+   https://github.com/larryhastings/gilectomy/tree/gilectomy.
+
+.. [#pyparallel] PyParallel.
+   http://pyparallel.org/.
+
+.. [#pythonsafethread] python-safethread.
+   https://launchpad.net/python-safethread
+
+.. [#gsteinpatch] https://www.python.org/ftp/python/contrib-09-Dec-1999/System/threading.tar.gz.
+
+.. [#dabeaz] An Inside Look at the GIL Removal Patch of Lore.
+   David Beazley. 2011.
+   https://dabeaz.blogspot.com/2011/08/inside-look-at-gil-removal-patch-of.html.
+
+.. [#jython] Jython.
+   https://www.jython.org/
+
+.. [#ironpython] IronPython.
+   https://ironpython.net/
+
+.. [#pypystm] PyPy: Software Transactional Memory.
+   https://doc.pypy.org/en/latest/stm.html
+
+
+
+Acknowledgments
+===============
+
+Thanks to Hugh Leather, Łukasz Langa, and Eric Snow for providing
+feedback on drafts of this PEP.
+
+Copyright
+=========
+
+This document is placed in the public domain or under the
+CC0-1.0-Universal license, whichever is more permissive.
+
+
+
+..
+    Local Variables:
+    mode: indented-text
+    indent-tabs-mode: nil
+    sentence-end-double-space: t
+    fill-column: 70
+    coding: utf-8
+    End:
```

Comment:
We don't need this bit any more (it's no longer in the template https://github.com/python/peps/blob/main/pep-0012.rst)

```suggestion
```